# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../nvidia/module-install-26.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Do not call depmod (especially not without the currently building version
attached).

  - Rene Rebe <rene@exactcode.de>

--- NVIDIA-Linux-x86-1.0-6629-pkg1/usr/src/nv/Makefile.kbuild.orig	2004-11-03 22:53:00.000000000 +0100
+++ NVIDIA-Linux-x86-1.0-6629-pkg1/usr/src/nv/Makefile.kbuild	2005-01-05 23:09:02.238876496 +0100
@@ -286,7 +286,7 @@
 module-install: suser-sanity-check module
 	@mkdir -p $(MODULE_ROOT)/video; \
 	install -m 0664 -o root -g root $(MODULE_OBJECT) $(MODULE_ROOT)/video; \
-	PATH="$(PATH):/bin:/sbin" depmod -ae;
+	echo "$(MODULE_OBJECT) installed successfully.";
 
 #
 # This target builds, then installs, then creates device nodes and inserts
